### ISSUE TITLE
published/non-pub flag on posts

### ISSUE THREAD

**Issue by meddlin** [meddlin/rushinglabs-blog#99]
Regular blog posts need a `"published": true/false` flag. ~~The docs section already has this.~~

## How should this work?

When a post's `frontmatter` metadata has `published: true` show it; when `published: false` don't show it.

```yaml
---
date: "2022-08-14"
title: Github's Tools for Managing a Repo
section: software
published: true
preview: |- 
    Wow!...GitHub has a lot of tools for managing repos!
---
```

```yaml
---
date: "2022-08-14"
title: Github's Tools for Managing a Repo
section: software
published: false
preview: |- 
    Wow!...GitHub has a lot of tools for managing repos!
---
```

**Comment by meddlin** [comment]
## Correction: The docs section _**does not**_ have this.

The projects page did. But nevertheless (haha, fun word) the functionality was added, and this should likely be put in the `/docs` section, too.

**Comment by meddlin** [comment]
Resolved by #104
